packages/define-stylex/src/core/index.ts

```diff
@@ -0,0 +1,119 @@
+import {
+  addNormalScript,
+  DEFINE_STYLEX,
+  generateTransform,
+  isCallOf,
+  MagicStringAST,
+  parseSFC,
+  walkAST,
+  type CodeTransform,
+} from '@vue-macros/common'
+import {
+  createTransformContext,
+  NodeTypes,
+  traverseNode,
+  type DirectiveNode,
+  type NodeTransform,
+} from '@vue/compiler-dom'
+import type { Node } from '@babel/types'
+
+const STYLEX_CREATE = '_stylex_create'
+const STYLEX_ATTRS = '_stylex_attrs'
+
+export function transformDirective(s?: MagicStringAST): NodeTransform {
+  return (node) => {
+    if (!(node.type === NodeTypes.ELEMENT)) return
```

Comment:
```suggestion
    if (!(node.type === (1 satisfies NodeTypes.ELEMENT))) return
```

Let's use `satisfies` expression instead of importing runtime enum, as they're not exported in lower Vue versions.

Reply:
Actually it's used more than one line.